wallet-webapps-common/src/main/webapp/vue-app/components/WalletNotificationAlert.vue

```diff
@@ -1,11 +1,15 @@
 <template>
   <v-alert
+    id="WalletNotificationAlert"
     v-model="displayAlert"
     :type="alertType"
-    class="paragraph"
+    :icon="alertType === 'warning' ? 'mdi-alert-circle' : ''"
```

Comment:
```suggestion
    :icon="alertType === 'warning' ? 'mdi-alert-circle' : ''"
```
we should use default icons as used in re-usable alert component defined in https://github.com/Meeds-io/social/blob/98038c5d5f086a6149b0be11b526584a320f3cb4/webapp/portlet/src/main/webapp/vue-apps/common/components/ExoNotificationAlert.vue#L4